crates/cext/src/transpiler/passes/basis_translator.rs

```diff
@@ -0,0 +1,87 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2025
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+use crate::pointers::mut_ptr_as_ref;
+use qiskit_circuit::circuit_data::CircuitData;
+use qiskit_circuit::converters::dag_to_circuit;
+use qiskit_circuit::dag_circuit::DAGCircuit;
+use qiskit_transpiler::passes::run_basis_translator;
+use qiskit_transpiler::standard_equivalence_library::generate_standard_equivalence_library;
+use qiskit_transpiler::target::Target;
+
+/// @ingroup QkTranspilerPasses
+/// Run the BasisTranslator transpiler pass on a circuit.
+///
+/// The BasisTranslator transpiler pass translates gates to a target basis by
+/// searching for a set of translations from the standard EquivalenceLibrary.
+///
+/// @param circuit A pointer to the circuit to run BasisTranslator on
+/// @param min_qubits The minimum number of qubits for operations in the input
+/// ciruit to translate.
+/// @param target The target where we will obtain basis gates from.
+///
+/// @returns A pointer to the resulting circuit from running the pass. If no
+/// modifications are made, it will return a pointer to the same circuit it was
+/// provided. Otherwise it will point to a new circuit instance.
+///
+/// # Example
+///
+/// ```c
+///    #include <qiskit.h>
+///
+///    QkCircuit *circuit = qk_circuit_new(3, 0);
+///    qk_circuit_gate(circuit, QkGate_CCX, (uint32_t[3]){0, 1, 2}, NULL);
+///
+///    // Create Target compatible with only U gates, with global props.
```

Comment:
Is this comment out of date, since it's mentioning U? How about just:
```suggestion
///    // Create a Target with global properties.
```

Reply:
Added in 2083432f29b7331be7002123a0c1d4b1206260b6